corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -435,6 +437,26 @@ def last_sync_log(self):
         else:
             return None
 
+    def _check_for_date_opened_bug(self, sync_log):
+        introduced_date = datetime(2016, 7, 19, 19, 15)
+        reverted_date = datetime(2016, 7, 20, 9, 15)  # date bug was reverted on HQ
+        resolved_date = datetime(2016, 7, 21, 0, 0)  # approximate date this fix was deployed
```

Comment:
we picked tonight at midnight as a safe guess for when this would get deployed. If this branch gets deployed after that, this will need to be updated.